base/dates/Dates.jl

```diff
@@ -1,5 +1,21 @@
 # This file is a part of Julia. License is MIT: http://julialang.org/license
 
+"""
+The Dates module provides `Date`, `DateTime`, `Time` types, and related funtions.
+
+The types are *timezone-unaware*, based on UT seconds (1/86400 of a day),
+and the proleptic Gregorian calendar, as specified in ISO 8601.
+For time zone functionality, see the TimeZones.jl package.
+
+Try
```

Comment:
Would be better to turn this into a doctests example block, see http://julia.readthedocs.io/en/latest/manual/documentation/.